packages/compute-baseline/src/browser-compat-data/supportStatements.ts

```diff
@@ -140,17 +138,113 @@ export class RealSupportStatement extends SupportStatement {
     }
   }
 
-  get version_added() {
+  get version_added(): string | false {
     return super.version_added as string | false;
   }
 
-  get version_removed() {
-    return super.version_removed as string | false;
+  get version_removed(): string | false | undefined {
+    return super.version_removed as string | false | undefined;
+  }
+
+  /**
+   * Find out whether this support statement says a given browser release is
+   * supported (with or without qualifications), unsupported, or unknown.
+   */
+  supportedIn(release: Release): Supported | Unsupported | UnknownSupport {
```

Comment:
@foolip I've refactored this as a method that just checks whether one release is supported. If one still wants to know the situation for every release, then that's a one-liner away:

`new Map(browser.releases.map(r => [r, statement.supportedIn(r)]))`

Also, doing this for any arbitrary support statement, while possible, was getting very tricky and hard to follow. Since _we_ don't need this I decided it wasn't worth the effort, so it's only available on `RealSupportStatement`s.

(I've kept some test cases and work from the all-values version, but it's not urgent, so I'll re-introduce that when there's an actual need or a spare couple of hours for it.)